tests/test_initialization.py

```diff
@@ -1544,3 +1550,112 @@ def new_state_dict():
     model = PeftModel.from_pretrained(model, tmp_path)
     assert any(msg in str(w.message) for w in recwarn.list)
     assert any(missing_key in str(w.message) for w in recwarn.list)
+
+
+class TestEvaInitialization:
+    # Constants for test configuration
+    COSINE_SIMILARITY_THRESHOLD = 0.75
+    NUM_SEEDS = 3
+    BATCH_SIZE = 4
+    MAX_LENGTH = 512
+
+    @pytest.fixture
+    def tokenizer(self):
+        tokenizer = AutoTokenizer.from_pretrained("openai-community/gpt2")
+        tokenizer.pad_token = tokenizer.eos_token
+        return tokenizer
+
+    @pytest.fixture
+    def dataset(self, tokenizer):
+        dataset = load_dataset("Rowan/hellaswag", split="train")
```

Comment:
I'd rather not use this dataset here in the tests, as it's downloaded from an external source. Would the test also work with, for instance, `"ybelkada/english_quotes_copy"`, which already use in many other tests? Maybe that can also help speed up testing.

Reply:
done